### PR TITLE
Fix module version

### PR DESCRIPTION
**Pull request by niveathika**
## Purpose
$subject

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes
 - Ran FindSecurityBugs plugin and verified report? yes
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -27,7 +27,7 @@
     <groupId>org.wso2.ei</groupId>
     <artifactId>module-mongo</artifactId>
     <packaging>pom</packaging>
-    <version>0.10.0</version>
+    <version>0.9.0</version>
```
Comment (daneshk) [inline pom.xml]: why you downgrade the version?

File: mongodb/src/mongodb/Module.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ The Mongo DB connector allows you to connect to Mongo DB from Ballerina and perf
 
 |                             |       Version               |
 |:---------------------------:|:---------------------------:|
-| Ballerina Language          | 1.1.1                       |
+| Ballerina Language          | 1.1.x                       |
```
Comment (wggihan) [inline mongodb/src/mongodb/Module.md]:
```suggestion
| Ballerina Language          | 1.2.x                       |
```

File: mongo-utils/src/main/java/org/wso2/mongo/MongoDBConstants.java [hunk 3]
```diff
@@ -22,7 +22,7 @@
 public class MongoDBConstants {
     public static final String ORGANIZATION_NAME = "wso2";
```
Comment (wggihan) [inline mongo-utils/src/main/java/org/wso2/mongo/MongoDBConstants.java]: Don't we need to change this?

File: mongodb/Ballerina.lock [hunk 4]
```diff
@@ -1,9 +1,9 @@
 org_name = "wso2"
```
Comment (wggihan) [inline mongodb/Ballerina.lock]:
```suggestion
org_name = "ballerina"
```

File: pom.xml (hunk 1 above)
Comment (niveathika) [reply pom.xml]: The latest release is only 0.8.2, it seems it was mistakenly bumped to 0.10 in a past PR.

**Pull request merged**